ironic/resource_ironic_node_v1.go

```diff
@@ -111,6 +111,10 @@ func resourceNodeV1() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"network_data": {
+				Type:     schema.TypeMap,
```

Comment:
I seem to vaguely recall that terraform maps do not like really sophisticated nested types. Have you tested it with a real network data?

Reply:
We did today, and you are right, we need a Map of lists, which can't be represented it seems.

We need to do some further inspection and probably do a complete rework to match our requirements, without breaking to much here.

I will close this PR until we have found a working solution internally.